java/src/lib.rs

```diff
@@ -153,3 +153,24 @@ fn throw_java_exception(mut env: JNIEnv, message: String) {
         }
     };
 }
+
+#[no_mangle]
+pub extern "system" fn Java_glide_ffi_resolvers_ScriptResolver_storeScript<'local>(
+    mut env: JNIEnv<'local>,
+    _class: JClass<'local>,
+    code: JString,
+) -> JObject<'local> {
+    let str: String = env.get_string(&code).unwrap().into();
+    let hash = glide_core::scripts_container::add_script(&str);
+    JObject::from(env.new_string(hash).unwrap())
+}
+
+#[no_mangle]
+pub extern "system" fn Java_glide_ffi_resolvers_ScriptResolver_dropScript<'local>(
+    mut env: JNIEnv<'local>,
+    _class: JClass<'local>,
+    hash: JString,
+) {
+    let str: String = env.get_string(&hash).unwrap().into();
+    glide_core::scripts_container::remove_script(&str);
```

Comment:
nit: &str looks too similar to the type for string slices. I think it would be better if it was renamed to something a little more descriptive ("hash_string"?)